nikola/plugins/task/scale_images.py

```diff
@@ -45,6 +45,7 @@ def set_site(self, site):
 
     def process_tree(self, src, dst):
         """Process all images in a src tree and put the (possibly) rescaled images in the dst folder."""
+        thumb_fmt = self.kw['image_thumbnail_format']
         ignore = set(['.svn'])
```

Comment:
Not really related to this PR but WTF. Also, Thumbs.db special casing a few lines below makes little sense.